app/commands/user/challenges/featured_exercises_progress_48_in_24.rb

```diff
@@ -12,7 +12,7 @@ class User::Challenges::FeaturedExercisesProgress48In24
     { week: 5, slug: 'protein-translation', title: 'Protein Translation', featured_tracks: %w[fsharp crystal csharp], learning_opportunity: "" },
```

Comment:
This is a fun little exercise that lets you delve into recursion, pattern matching, higher order functions, and other functional concepts. Our featured solutions also use bitshifting, regexp and more!